ctl/src/command.rs

```diff
@@ -25,41 +27,61 @@ fn generate_tagged_id(tag: &str) -> String {
   format!("{}-{}", tag, s)
 }
 
+// Run the code waiting for messages in a separate thread. Just before finishing the thread sends a message.
+// The calling code waits for this message with a timeout.
+// Note: This macro is used only for simple command which has any/simple computing
+// to do with the message received.
+macro_rules! command_timeout {
+  (dur => $duration: expr, b => $block: expr) => (
```

Comment:
why this syntax for the macro's arguments?

Reply:
You're right, it's not necessary. I removed